### DISCUSSION TITLE
There is a large training error

### DISCUSSION THREAD

**Discussion by ChengQIAN93** [deepmodeling/deepmd-kit#726] [Q&A]
Hi，

 I try to use deepmd-kit to fit a Fe potential. There are some crystal structures, liquid structures, and surface structures in my dataset (The number of surface structures is less than the bulk). After training,  The model can describe the bulk structures well, but there is a large deviation for the surface structure. So could yosu give me some suggestions to train a better model?  

**This is for bulk** 
![result](https://user-images.githubusercontent.com/77376448/121488014-a2d22a00-ca0d-11eb-8040-0ac70163d2f3.png)

**These are for surface**
![Fe100](https://user-images.githubusercontent.com/77376448/121488110-b8dfea80-ca0d-11eb-83dc-e5824e35f9ea.png)
![Fe110](https://user-images.githubusercontent.com/77376448/121488120-bc737180-ca0d-11eb-81fc-e85a8ae2a9f3.png)
![Fe111](https://user-images.githubusercontent.com/77376448/121488126-bed5cb80-ca0d-11eb-89e7-fe7c1776800e.png)

**and This is the training input file**

![06b17557be2774a66dbe2ea3269beb6](https://user-images.githubusercontent.com/77376448/121488421-06f4ee00-ca0e-11eb-95b1-c94eb4eceac0.png)

Thank you

**Comment by AnguseZhang** [comment]
What's the source of your training data?
Did you check the data quality of surface data, such as convergence of first-principles?
